integration_tests/pom.xml

```diff
@@ -228,12 +228,24 @@
                     <version>${spark.version}</version>
                     <scope>provided</scope>
                 </dependency>
+                <dependency>
+                    <groupId>org.apache.spark</groupId>
+                    <artifactId>spark-unsafe_${scala.binary.version}</artifactId>
+                    <version>${spark.version}</version>
+                    <scope>provided</scope>
+                </dependency>
```

Comment:
somehow this list got out of sync with https://github.com/NVIDIA/spark-rapids/blob/branch-23.08/jenkins/databricks/install_deps.py#L50 . The reason that this change is sufficient is that it is already included in install_deps. Would be great to have the python script use the dedeps profile dependencies as the list of record to track